### DISCUSSION TITLE
Adding particles manually in checkpoint

### DISCUSSION THREAD

**Discussion by mojtabaAmiraslanpour** [UvaCsl/HemoCell#91] [General discussion]
Hi,

I am running a case with different numbers of platelets. I have this issue where I must run each case from scratch and wait for it to get steady. If it is possible to run only one case until the steady state and then add the platelets, I will save a huge amount of time and resources.

I see the results are stored in `particleField.dat` and `internalViscosity.dat` files. Any hints on how I can write my own custom files for these so that I can start my simulation from any checkpoint I need with the custom particle (plt) field?

**Comment by gzavo** [comment]
@mojtabaAmiraslanpour,

These are binary streams of data structures from HemoCell. I can't think of an easy way of writing these manually.
Also, it would probably not help, since the flow field is coupled with the cells. Changing the cells but not the fluid field would not yield anything steady.

**Comment by mojtabaAmiraslanpour** [comment]
@gzavo I am only thinking about adding platelets and not the whole cells. In this way, steadiness won't be affected much. If there is a way to store RBCs and PLTs separately, I may be able to overcome this by saving different checkpoints at different stages of a problem and combining them.
